tools/scripts/brokenLinkChecker/brokenLinkChecker.js

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env node
+
+const blc = require('broken-link-checker');
+const https = require('https');
+const url = require('url');
+
+var whitelist = require('./whitelist.json').ignore;
+
+const slack_url =
+  // test room URL
+  "https://hooks.slack.com/services/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";
+
+  // staging room URL
+  //"https://hooks.slack.com/services/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";
+
+function numberWithCommas(x) {
```

Comment:
worth a comment saying what this does. (my initial guess is takes something like "1000" and makes it "1,000"? If so, you might also just be able to do something like `(1000).toLocaleString()`.